src/prefect/tasks/mysql/mysql.py

```diff
@@ -173,13 +178,27 @@ def run(
                 "The 'fetch' parameter must be one of the following - ('one', 'many', 'all')"
             )
 
+        cursor_types = {
+            "cursor": pymysql.cursors.Cursor,
+            "dictcursor": pymysql.cursors.DictCursor
+        }
+
+        if callable(cursor_type):
```

Comment:
I believe @jcrist checks for a callable here because it's quick and does not limit cursors to things that explicitly  nherit from `Cursor` but I think it's probably worth doing this
```suggestion
        if isinstance(cursor_type, pymysql.cursors.Cursor):
```

Reply:
This could also just remain `callable` and we can let the pymysql library determine if it's a valid Cursor as it pleases

Reply:
I wouldn't check the types, minimal work on our end. I'd only validate `cursor_type` is in the set of string shorthand names, or is a callable. Let the mysql library error if needed in other cases.